dspace-api/src/test/data/dspaceFolder/assetstore/crossRef-test.json

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
Again, please move to `dspace-server-webapp/src/test/resources/org/dspace/external/provider/impl/` like mentioned in your other PRs.  Test files should always be placed alongside the tests that use them.